### ISSUE TITLE
Regression: Dolphin Pop

### ISSUE THREAD

**Issue by desertking** [ruffle-rs/ruffle#14719]
### Describe the bug

This game used to run fine. Now it shows a black screen after selecting a level. sound still running thou.

### Expected behavior

work again

### Content Location

https://www.spiele-umsonst.de/azad/downloads/flashgames/DolphinPop.swf

### Affected platform

Online demo

### Operating system

All

### Browser

All

**Comment by Lord-McSweeney** [comment]
The first error (at the title screen) is
```
Error dispatching event EventObject(EventObject { type: "enterFrame", class: flash.events::Event, ptr: 0x555f4ed367b0 }) to handler FunctionObject(FunctionObject { ptr: 0x555f4e6ae250, name: Ok("com.zlonggames.view::AbstractGeneralView/changeState()") }) : TypeError: Error #1009: Cannot access a property or method of a null object reference. (accessing field: http://adobe.com/AS3/2006/builtin::split)
	at com.zlonggames.control.coreconcrete.fromCoreApp.interfaces::TimerManage/initBestTime()
	at com.zlonggames.control.coreconcrete.fromCoreApp.interfaces::TimerManage/init()
	at com.zlonggames.view.level::LevelInGame/initDisplay()
	at com.zlonggames.view.level::LevelInGame()
	at com.zlonggames.view.menu::GatePassMenu/nextUIScene()
	at com.zlonggames.view::AbstractGeneralView/changeState()
```

**Comment by n0samu** [comment]
Another regression caused by #13966

**Comment by evilpie** [comment]
Fixed by #14801.
